lib/attach-middleware.js

```diff
@@ -34,4 +34,41 @@ module.exports = function(app, options) {
       });
     },
     logError);
+
+  function shouldExclude(moduleName, excludes) {
+    for (var i = 0; i < excludes.length; i++) {
+      if (minimatch(moduleName, excludes[i])) {
+        return true
+      }
+    }
+    return false
+  }
+  app.post('/filter-entries',bodyParser.urlencoded({
+    limit: '50mb',
+    extended: false,
+    parameterLimit: 10000
+  }),bodyParser.json({
+    limit: '50mb'
+  }),
+    function(req, res) {
+      var entries = req.body['entries[]'] || req.body.entries
+      debugger
+      var seen = {}
+      var appName = options.appName
+      var filteredEntries = entries.filter(function(entry) {
+        var result = false
+        if (typeof (seen[entry]) === 'undefined') {
+          var parts = entry.split('/')
+          if (parts.length >= 1 && parts[0] === appName || parts.length >= 2 && parts[0] === 'dummy' && parts[1] === appName) {
```

Comment:
I'm wary of this - @rwjblue is this a predictable/consistent format that can be relied on?